### PR TITLE
Fix more grammatical and typos

### PR DESCRIPTION
**Pull request by Excigma**
Fix more grammatical and typo~~e~~s that persisted through the other PRs. I, however, haven't attempted to make the flow nicer and make descriptions more succinct, removing redundant and long-winded ways of saying things, which may be possible in the future with another PR.

### CODE REVIEW

File: protocol/data.md [hunk 1]
```diff
@@ -112,16 +112,16 @@ inline void Encode(BinData &out) const {
...
+**Quick Misconception,** in Diep's console, sometimes you'll see messages relating to _"possible desyncs"_ and alongside them, you'll see two numbers inside of these `<triangular bracket things>`, these are NOT `<hash, id>`. Unlike the order they are read, entids are logged as `<id, hash>` in the Diep console.
 
 ---
 
 ### **Data Organization**
 
-Inside of a specifically 0x00 packets, there is a form of table which has come to be known as a `jumpTable`. Jump tables work differently than the standard arrays, by retrieving indexes from jumps, then parsing values based on index. The best way to explain the format is with an example, here is an example of a field group identification table, without any values.
+Inside of specifically 0x00 packets, there is a form of table which has come to be known as a "jump table. Jump tables work differently than the standard arrays, by retrieving indexes from jumps, then parsing values based on that index. The best way to explain the format is with an example, here is an example of a field group identification table, without any values.
```
Comment (Excigma) [inline protocol/data.md:124]: I am not sure if this paragraph here makes sense

**Review by Excigma — COMMENTED** [review]
Maybe someone else needs to check this, idk.

File: protocol/data.md (hunk 1 above)
Comment (ABCxFF) [reply protocol/data.md:124]: maybe rephrase to
```
Inside of 0x00 packets, there is data stored in a table-like form of data, which has come to be known as a "jump table". Jump tables work differently than standard arrays. They are parsed by retrieving indexes from jumps, then parsing values based on that index. The best way to explain the format is with an example, here is an example of a field group identification jump table, in this case, indexes without any values.
```

Comment (Excigma) [reply protocol/data.md]: Rephrased

**Comment by Excigma** [comment]
It's also worth noting that there is inconsistent use of both "api" and "API", which I haven't fixed.

**Review by ABCxFF — APPROVED** [review]

**Comment by ABCxFF** [comment]
i should read it over again

**Comment by binary-person** [comment]
perhaps we should also type out a documentation convention readme for future collaborators? For example, Diep units, not diep units. API, not api. TCP, not tcp, etc...

**Review by endlessXD — DISMISSED** [review]
nvm

**Comment by binary-person** [comment]
@ABCxFF merge?

**Review by binary-person — APPROVED** [review]
